numpy/testing/nosetester.py

```diff
@@ -413,27 +402,55 @@ def test(self, label='fast', verbose=1, extra_argv=None,
             for warningtype in raise_warnings:
                 warnings.filterwarnings('error', category=warningtype)
             # Filter out annoying import messages.
-            warnings.filterwarnings('ignore', message='Not importing directory')
-            warnings.filterwarnings("ignore", message="numpy.dtype size changed")
-            warnings.filterwarnings("ignore", message="numpy.ufunc size changed")
-            warnings.filterwarnings("ignore", category=np.ModuleDeprecationWarning)
-            warnings.filterwarnings("ignore", category=FutureWarning)
```

Comment:
Ah, I was thinking about the boolean negative stuff, but that I did not remove yet (and I think I can just leave it for now). I will see if I manage to test scipy dev with this branch.